tests/integration/test_tiler.py

```diff
@@ -233,7 +260,7 @@ def it_locates_tiles_on_the_slide(
         )
 
         # --- Expanding test report with actual and expected images ---
-        expand_tests_report(request, expected=expected_img, actual=tiles_location_img)
+        # expand_tests_report(request, expected=expected_img, actual=tiles_location_img)  # noqa: E800, E501
```

Comment:
is this intentional commented out?

Reply:
I remember the CI breaking with this uncommented, don't know why. I'll retry